app/src/main/java/com/example/admin/lab_56_57/MainActivity.java

```diff
@@ -0,0 +1,71 @@
+package com.example.admin.lab_56_57;
+
+import android.app.Activity;
+import android.app.ListActivity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.ActionBarActivity;
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.ListView;
+import android.widget.Spinner;
+
+
+public class MainActivity extends AppCompatActivity {
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        // Button for launching activity for Lab #56
+        Button launcher56 = (Button) findViewById(R.id.launcher56);
+
+        launcher56.setOnClickListener(new View.OnClickListener() {
+                                          @Override
```

Comment:
https://source.android.com/source/code-style.html. Paragraph "Use Spaces for Indentation"